### PR TITLE
database + styling changes

### PR DESCRIPTION
**Pull request by Capn05**
I fixed issue where newly registered users weren't being added to the database. Added completed css styling for the login and signup.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -6,7 +6,7 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Flixster</title>
   </head>
-  <body style="background-color: white;">
+  <body style="background-color: #111">
```
Comment (BriptimusPrimus) [inline index.html:9]: Can you add screenshots to the PR description?

File: src/login/login.jsx [hunk 2]
```diff
@@ -25,52 +25,50 @@ const Login = () => {
   };
 
   return (
-    <div>
-      {userLoggedIn && <Navigate to={"/home"} replace={true} />}
-      <main>
+    <div className="loginContainer">
+      <div className="center">
```
Comment (BriptimusPrimus) [inline src/login/login.jsx:29]: Does `<div className="loginContainer center">` works?
It's always better to have less divs, so we try minimizing them when possible

File: src/login/login.jsx [hunk 3]
```diff
@@ -25,52 +25,50 @@ const Login = () => {
...
+            <input
+              type="text"
+              autoComplete="email"
+              required
+              value={email}
+              onChange={(e) => {
+                setEmail(e.target.value);
+              }}
+            />
+            <span></span>
```
Comment (BriptimusPrimus) [inline src/login/login.jsx:45]: What is this span for? How does it the component look like if removed?

Comment (Capn05) [reply src/login/login.jsx:45]: I think I was using it for something and I forgot to delete it

Comment (Capn05) [reply src/login/login.jsx:45]: I incorporated the span into the css check login.css for more details

**Pull request merged**